ci: add GH Actions dependabot

endersonmaia opened the pull request:
This PR will add a CI job to check for updates on actions used on the CI.

You can see an example PR here: https://github.com/sunodo/sunodo/pull/189

marcelstanley commented on CHANGELOG.md:
```diff
@@ -297,7 +303,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [0.4.0]: https://github.com/cartesi/rollups/releases/tag/v0.4.0
 [0.3.0]: https://github.com/cartesi/rollups/releases/tag/v0.3.0
 [0.2.0]: https://github.com/cartesi/rollups/releases/tag/v0.2.0
-[0.1.3]: https://github.com/cartesi/rollups/releases/tag/v0.1.3
-[0.1.2]: https://github.com/cartesi/rollups/releases/tag/v0.1.2
-[0.1.1]: https://github.com/cartesi/rollups/releases/tag/v0.1.1
-[0.1.0]: https://github.com/cartesi/rollups/releases/tag/v0.1.0
```
Any reason for deleting the `Unreleased` and `pre-0.2.0` releases from the list?

endersonmaia replied:
No reason, I'll put those back.

endersonmaia replied:
Actually, my VSCode is removing those when I save the file.

endersonmaia replied:
That was the reason: https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md053.md

marcelstanley commented on CHANGELOG.md:
```diff
@@ -30,8 +36,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed `advance-runner`, `dispatcher`, `graphql-server`, `host-runner`, `inspect-server`, `indexer`, and `state-server` docker images
 
 ### Fixed
+
```
Linter execution detected. Should we add a GH action for that?

gligneul commented:
For some reason, the CI is not working correctly for PR made from forks. I changed the base to a branch called [feature/add-gh-actions-dependabot](https://github.com/cartesi/rollups-node/tree/feature/add-gh-actions-dependabot) so we can merge there and check if the CI is passing before merging it to main.

@endersonmaia, I also added you as a writer for the repo; now, you should be able to create branches without the fork.

endersonmaia commented:
> For some reason, the CI is not working correctly for PR made from forks. I changed the base to a branch called [feature/add-gh-actions-dependabot](https://github.com/cartesi/rollups-node/tree/feature/add-gh-actions-dependabot) so we can merge there and check if the CI is passing before merging it to main.
> 

If the CI uses some Secrets, they'll fail.

> @endersonmaia, I also added you as a writer for the repo; now, you should be able to create branches without the fork.

:+1:

endersonmaia commented:
moved to PR https://github.com/cartesi/rollups-node/pull/120
